bpfd/Cargo.toml

```diff
@@ -41,3 +41,6 @@ tokio-stream = { version = "0.1.12", features = ["net"] }
 sha2 = "0.10.6"
 base16ct = { version = "0.2.0", features = ["alloc"] }
 tempfile = "3.4.0"
+
+[dev-dependencies]
+rstest = "0.17.0"
```

Comment:
👀

Reply:
Seemed like a good crate for easily writing multiple test cases I can use something else if you'd like!